zebra-consensus/src/verify.rs

```diff
@@ -268,11 +266,8 @@ mod tests {
             .await;
 
         ensure!(
-            match verify_result {
-                Ok(_) => false,
-                // TODO(teor || jlusby): check error string
-                _ => true,
-            },
+            // TODO(teor || jlusby): check error string
```

Comment:
I wouldn't default to doing string comparisons, if there's ever a specific error that we need to check for that is being constructed from `str -> Box<dyn Error>` we should instead change the error to be a proper error type that we can downcast too, using string's content as a proxy for a typed error is a recipe for disaster.

Reply:
I'll fix these comments, possibly in a future PR.

Reply:
Fixed in #538 in 5f96b3239843790f498cbcdd45ad556db949b6ae for verify/block

Reply:
Fixed in #538 in 609de233128140a92f4866c67f847c24f4dd192c for checkpoint